resources/pelias/dictionaries/libpostal/en/stopwords.txt

```diff
@@ -0,0 +1,2 @@
+!in
```

Comment:
hmm.. this `!in` doesn't need to be here, I'l probably remove this as the tests pass without it.

Reply:
Ugh, so I think I'll split this out into a separate PR in case it causes any issues.

Reply:
Added test cases and restored the `!in` line, closed issue.

```
  // https://github.com/pelias/parser/issues/191
  assert('Indianapolis, IN', [
    { locality: 'Indianapolis' },
    { region: 'IN' }
  ])
  assert('1441 Brown Ave, Whiting, IN 46394', [
    { housenumber: '1441' },
    { street: 'Brown Ave' },
    { locality: 'Whiting' },
    { region: 'IN' },
    { postcode: '46394' }
  ])
 ```